src/js/components/molecules/cards/card-download/card-download.ts

```diff
@@ -1,6 +1,7 @@
 import { Vue, Component, Prop } from 'vue-property-decorator';
 import { CardDownloadProps } from './card-download.models';
 import WithRender from './card-download.vue.html';
+import v5 from '../../../../utils/uuid';
```

Comment:
```utils/uuid``` should work

Reply:
Yes.

(This does too, but without magic)